src/dgipy/dgidb.py

```diff
@@ -220,7 +220,78 @@ def get_drug_applications(
     return result
 
 
-def __process_drug(results: dict) -> pd.DataFrame:
+def get_clinical_trials(
+    terms: str | list,
+) -> pd.DataFrame:  # TODO: Better error handling for new_row?, use_pandas=False
+    """Perform a look up for clinical trials data for drug or drugs of interest
+
+    :param terms: drug or drugs of interest
+    :return: all clinical trials data for drugs of interest in a DataFrame
+    """
+    base_url = "https://clinicaltrials.gov/api/v2/studies?format=json"
+    rows_list = []
+
+    if isinstance(terms, str):
+        terms = [terms]
```

Comment:
since the arg is named `terms` it should probably just be a list -- a user can submit a single-item list if they really want to

Reply:
Oh wait we do this everwyhere. Change in separate issue